corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py

```diff
@@ -0,0 +1,48 @@
+from corehq.apps.sms.mixin import BackendMapping
```

Comment:
@biyeun yeah i was probably going to run it tomorrow at some point, but there's no particular urgency since all the code still is working off of the couch models. why do you ask?